quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/Build.java

```diff
@@ -84,6 +92,16 @@ public void run() {
         }
     }
 
+    private static void configureBuildClassLoader() {
+        ConfigValue ignoredArtifacts = Configuration.getCurrentBuiltTimeProperty(
+                MicroProfileConfigProvider.NS_KEYCLOAK_PREFIX + ClassLoaderOptions.IGNORE_ARTIFACTS.getKey());
+
+        if (ignoredArtifacts != null && StringUtil.isNotBlank(ignoredArtifacts.getValue())) {
+            // ignored artifacts must be set prior to starting re-augmentation
+            System.setProperty("quarkus.class-loading.removed-artifacts", ignoredArtifacts.getValue());
+        }
```

Comment:
Just out of curiosity, why do we need this if we already have mapping here:
https://github.com/keycloak/keycloak/blob/563b37934373d4eb8422a0aa0ca3ca4100455ca1/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/ClassLoaderPropertyMappers.java#L25-L32
Can't we remove either of them?

Reply:
@vmuzikar To leverage the related fix from Quarkus, we need to make sure the `build` class loader is built taking this property into account so that the artifacts are ignored accordingly.

By keeping only within our mapping layer, it is too late because the `build` class loader was already built.

We could potentially keep everything within the `Build` command but I did not test it. Also, I think keeping the resolution within our configuration is more natural and aligned with what we are doing for others.